docs/user/glossary.rst

```diff
@@ -53,13 +53,13 @@ Forbidden translations
 
 .. versionadded:: 4.5
 
-You can flag certain glossary terms as forbidden - those should not be used
-while translating. Use this to clarify translation when some words are
+You can flag certain glossary terms as forbidden, meaning ones _not_ to be used
+for translations. Use this to clarify translation when some words are
 ambiguous or could have unexpected meanings.
 
 The terms can be flagged on the source language using :guilabel:`Tools` ↓
```

Comment:
```suggestion
The terms can be flagged using :guilabel:`Tools` ↓
```

The source language limitation will be gone here with https://github.com/WeblateOrg/weblate/pull/5399

Reply:
As I can see it still requires `edit_source` permission, which most users don't have. Which should be mentioned unless it will be changed soon.

Also won't toggling the flag affect the translation of the term to all the languages? Unless it's a bug it should be mentioned too...

Reply:
No, it applies it for single language only ... and I will adjust the permission check in this case as well